internal/core/src/segcore/ChunkedSegmentSealedImpl.cpp

```diff
@@ -883,15 +876,42 @@ ChunkedSegmentSealedImpl::DropIndex(const FieldId field_id) {
                "Field id:" + std::to_string(field_id.get()) +
                    " isn't one of system type when drop index");
     auto& field_meta = schema_->operator[](field_id);
-    AssertInfo(field_meta.is_vector(),
-               "Field meta of offset:" + std::to_string(field_id.get()) +
-                   " is not vector type");
+    AssertInfo(!field_meta.is_vector(), "vector field cannot drop index");
```

Comment:
Before this PR, vector index can be dropped while scalar cannot, but after this PR, they are reversed?